Algolia.Search/AlgoliaClient.cs

```diff
@@ -315,6 +315,7 @@ public JObject DeleteIndex(string indexName, RequestOptions requestOptions)
         /// <param name="dstIndexName">The new index name that will contain a copy of srcIndexName (destination will be overriten if it already exists).</param>
         /// <param name="requestOptions"></param>
         /// <param name="token"></param>
+        [Obsolete("Client.MoveIndex is deprecated, please use Index.MoveTo instead")]

```

Comment:
`s/Index.MoveTo/Index.MoveToAsync/` I think

Reply:
`s/Client.MoveIndex/Client.MoveIndexAsync` as well

Reply:
Fixed in [877139b](https://github.com/algolia/algoliasearch-client-csharp/pull/523/commits/877139bb35c630467a336c2fb369830f156e4d4c)